_positions/visual-designer.md

```diff
@@ -1,40 +1,29 @@
 ---
 title: Visual Designer
-permalink: who-we-are-hiring/positions/visual-designer/
+permalink: roles-and-teams/positions/visual-designer/
 team: design
 active: false
 ---
 
-The Visual Designer starts with a deep understanding of the goals of
-customers and the business so that they can create experiences that
-delight. Visual Designers will be well-versed in all aspects of current
-visual design standards and trends and will be responsible for managing
-project design reviews, resource planning, and execution for all project
-work related to visual design.
-
-Primarily responsible for:
-
--   Oversees all visual design efforts
-
--   Guides, mentors, and coaches team members while leading projects to
-successful completion
-
--   Develops and maintains relationships with key peers in Marketing,
-Branding, UX leaders, IT leaders, and others to identify and plan
-creative solutions
-
--   Manages external service resources and budgets for visual design
-
--   Ensures successful completion of all work executed by the team (on
-time, on budget, and ensuring quality)
-
--   Ensures compliance with the project management methodologies and the
-Project Management Office processes and standards
-
--   Develops, maintains, and ensures compliance of application release
-management, outage management and change control processes and
-standards
-
--   Defines, creates, communicates, and manages resource plans and other
-required project documentation such as style guides and provides
-updates as necessary
+##Role: Visual Designer
+
+###Position Summary
```

Comment:
Please change capitalization to 'Position summary' for consistency with other headings on the site.